build/tutorials/tutorials-contest/drizzle-auction-manager/frontend/Auction.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState, useEffect } from "react";
```

Comment:
Port .js -> .ts

Reply:
Hi, @cgcardona. Thanks for the review. I have tried to do this, but it is quite difficult to port to Typescript. Migrating to .ts breaks a lot of code and gives compilation errors, like the one shown below. Same for the other two tutorials (**E-Voting DApp** and **Distributed File Manager**).

![Screenshot from 2021-09-14 01-17-17](https://user-images.githubusercontent.com/40765810/133147820-eca72dc8-e7a8-4e56-9a05-ef5d67b8c3bd.png)